test/unit/runner.spec.js

```diff
@@ -544,6 +552,9 @@ describe('Runner', function() {
       it('should not hang if overlong error message is single line', function(done) {
         var hook = new Hook();
         hook.parent = suite;
+        var data = [];
+        // mock a long message
+        for (var i = 0; i < 10000; i++) data[i] = {a: 1};
```

Comment:
These lines should not be present.
`genOverlongSingleLineMessage()` handles this...